modules/imgproc/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ ocv_add_dispatched_file(median_blur SSE2 SSE4_1 AVX2)
 ocv_add_dispatched_file(morph SSE2 SSE4_1 AVX2)
 ocv_add_dispatched_file(smooth SSE2 SSE4_1 AVX2)
 ocv_add_dispatched_file(sumpixels SSE2 AVX2 AVX512_SKX)
+ocv_add_dispatched_file(warp_kernels SSE2 SSE4_1 AVX2 NEON RVV LASX)
```

Comment:
what about NEON_FP16?

Reply:
Added. But actually `FP16` is set to be one of the baselines according to the CMake log:

```
--   CPU/HW features:
--     Baseline:                    NEON FP16
--     Dispatched code generation:  NEON_DOTPROD NEON_FP16 NEON_BF16
--       requested:                 NEON_FP16 NEON_BF16 NEON_DOTPROD
--       NEON_DOTPROD (1 files):    + NEON_DOTPROD
--       NEON_FP16 (3 files):       + NEON_FP16
--       NEON_BF16 (0 files):       + NEON_BF16
```

`FP16` enables `NEON_FP16`.